player/lua/stats.lua

```diff
@@ -45,15 +45,15 @@ local o = {
     plot_color = "FFFFFF",
 
     -- Text style
-    font = "sans-serif",
+    font = "",
```

Comment:
```suggestion
    font = nil,
```
and similar for other cases.

Reply:
`read_options` docs say: "A table with key-value pairs. The type of the default values is important for converting the values read from the config file or command-line back. Do not use nil as a default value!"

Reply:
meh

Reply:
You could sanitize empty string to `nil` after the call to `read_options`